Data Science Tools and Ecosystem

In this notebook, Data Science Tools and Ecosystem are summarized.

__Objetives:__
- Categories of Data Science Tools
- Languages of Data Science
- Packages, APIS, Data Sets, and Models
- Jupyter Notebooks and JupyterLAb

Some languages for Data Scientists are:

1. "R" language
2. Python
3. JavaScript
4. C++

Some of the commonly used libraries used by Data Scientists include:

1. MySQL
2. IBM Db2
3. Microsoft SQL Server

| Data Science Tools |
| --- |
| Jupyter Notebooks  |
| R Studio  |
| GitHub  |

### Below are a few examples of evaluating arithmetic expressions in Python.

In [14]:
(3*4)+5

17

"This a simple arithmetic expression to mutiply then add integers"

"This will convert 200 minutes to hours by diving by 60"

In [13]:
200 / 60

3.3333333333333335

R= There are 3.3 hours on 200 minutes.

## Author ##
Irving Peña Ferreiro